### PR TITLE
Update alternative-text.md

### PR DESCRIPTION
**Pull request by raquelbreternitz**
Added some guidance on punctuation and a link from Scott Vinkle on the Shopify blog.

Would love thoughts on wording.

### CODE REVIEW

File: polaris.shopify.com/content/content/alternative-text.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Always write alt text in plain text. The average rate of listening to a screen r
 - Avoid needless words. If a phrase is still understandable without a specific word, remove it.
 - Remove articles like "a, an, one of," etc. whenever possible. Alt text has different grammatical rules. "Filler words" that assist understanding in speech can get in the way in alt text.
 - Avoid using "image of" or "photograph" unless the type of image is relevant to the context. Screenreaders already announce images with use of the `<img>` attribute.
-- Avoid punctuation like `!!` and emoji like 🥰. Screen readers will announce these as "exclamation point, exclamation point" and "smiling face with three hearts."
+- Avoid punctuation like `!!` and emoji like 🥰. Screen readers will announce these as "exclamation point, exclamation point" and "smiling face with three hearts." Do use other punctuation marks as you regularly would, such as commas or periods at the ends of the sentences. The screen-reader will react with a brief pause.
```
Comment (jjgali) [inline polaris.shopify.com/content/content/alternative-text.md]: Suggestion:
>such as commas, or a period at the end of a sentence

The plural of periods/ends/sentences gets a little confusing, I think. 
And although it's not required, I think a comma after "such as commas" would help clarify that commas don't go at the end of a sentence—just periods.

File: polaris.shopify.com/content/content/alternative-text.md [hunk 2]
```diff
@@ -242,7 +242,7 @@ When accounting for SEO in your alt text:
 ## Resources
 
 For more information on coding and alt text standards, visit the
-[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/intro/wcag.php) or [WebAIM Alternative Text Guidelines](https://webaim.org/techniques/alttext/).
+[Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/intro/wcag.php) or [WebAIM Alternative Text Guidelines](https://webaim.org/techniques/alttext/). Read some more [considerations when writing alt text](https://ux.shopify.com/considerations-when-writing-alt-text-a9c1985a8204) from our accessibility specialist.
```
Comment (jjgali) [inline polaris.shopify.com/content/content/alternative-text.md]: Suggestion: 
>For detailed guidance on coding and alt text standards, visit….You can also read some more considerations when writing alt text from a Shopify accessibility specialist.

The two "mores" in that paragraph threw me a little.

**Review by jjgali — APPROVED** [review]
TY @raquelbreternitz !

File: polaris.shopify.com/content/content/alternative-text.md (hunk 1 above)
Comment (raquelbreternitz) [reply polaris.shopify.com/content/content/alternative-text.md]: Excellent, added both suggestions, thank you!

**Pull request merged**